src/ui/server/helpers/data-content-mappings/map-month-string.ts

```diff
@@ -1,15 +1,15 @@
 /**
- * mapPeriodMonths
+ * mapMonthString
  * Map credit period answer to a month/s string
  * @param {Number} Credit period
  * @returns {String} Credit period with month/s string
  */
-const mapPeriodMonths = (answer: number) => {
+const mapMonthString = (answer: number) => {
```

Comment:
could answer ever be 0?

Reply:
Potentially, in which case it'd return `0 months`